docs/user_guide.md

```diff
@@ -701,18 +672,7 @@ void test() {
   --export-source-suppress
```

Comment:
Do we want to keep this? Shouldn't we have something like export user data, or export issue data instead?